test/TellorPriceOracle.t.sol

```diff
@@ -4,17 +4,17 @@ pragma solidity 0.8.19;
 import { Test } from "forge-std/Test.sol";
 import { ITellor } from "../contracts/Dependencies/ITellor.sol";
 import { IPriceOracle } from "../contracts/Oracles/Interfaces/IPriceOracle.sol";
-import { TellorPriceOracleWETH } from "../contracts/Oracles/TellorPriceOracleWETH.sol";
+import { TellorPriceOracle } from "../contracts/Oracles/TellorPriceOracle.sol";
 
-contract TellorPriceOracleWETHTest is Test {
+contract TellorPriceOracleTest is Test {
```

Comment:
Same as for Chainlink ("I wouldn't rename this, how and for other coins, we could add tests to see is setup good.")